sdk/src/lib.rs

```diff
@@ -41,3 +47,116 @@ pub enum OxideAuthError {
     #[error("no authenticated hosts; use oxide auth login to authenticate")]
     NoAuthenticatedHosts,
 }
+
+impl progenitor_client::ClientHooks for Client {
```

Comment:
Can you please add a comment about why we're implementing this?

Reply:
Done.